cfpb/consumerfinance.gov
# Retirement: Hide Tool - Spanish Translation - Clean

**This is a follow-up on https://github.com/cfpb/consumerfinance.gov/pull/7929 - a warning/alert field on the [before-you-claim](https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/) page**

**To test:**

- navigate to: http://localhost:8000/consumer-tools/retirement/before-you-claim
- disable javascript
- reload page and verify warning field is shown and retirement tool is hidden
- click "Ver página en español"
- verify the text has been translated to Spanish
- text should read: "Este sitio no es compatible con su navegador web. Intente con un navegador web más nuevo o confirme que JavaScript está habilitado."

![Screenshot 2023-08-18 at 5 04 07 PM](https://github.com/cfpb/consumerfinance.gov/assets/130792942/e6e4fbc6-3601-4b16-8bd5-05f4055b5dfe)
